docs/benchmarking.md

```diff
@@ -115,6 +115,67 @@ TODO(benvanik): Talk about VM Benchmarks
 IREE is instrumented with the C++ bindings from the
 [Google Web Tracing Framework](https://github.com/google/tracing-framework).
 
+To use tracing in IREE, you need build IREE with following requirements: 1) Turn
+`IREE_ENABLE_RUNTIME_TRACING` on. 2) Add `-DNDEBUG` to `IREE_DEFAULT_COPTS`.
+
+For example:
+
+```shell
+export IREE_DEFAULT_COPTS='-DNDEBUG'
+cmake -B build/ \
```

Comment:
Won't this be a debug build? Probably want RelWithDebInfo

Reply:
@hanhanW

Reply:
Oh yes, fixed. (also add the third requirement above).